builtin/providers/aws/resource_aws_subnet.go

```diff
@@ -65,16 +66,16 @@ func resourceAwsSubnetCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	// Get the ID and store it
-	subnet := &resp.Subnet
-	d.SetId(subnet.SubnetId)
-	log.Printf("[INFO] Subnet ID: %s", subnet.SubnetId)
+	subnet := resp.Subnet
+	d.SetId(*subnet.SubnetID)
+	log.Printf("[INFO] Subnet ID: %s", subnet.SubnetID)
```

Comment:
This will output something like `aws.StringValue)(0xc20814ebe0)` unless you dereference it, instead can you do:

    log.Printf("[INFO] Subnet ID: %s", *subnet.SubnetID)

Like https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_subnet.go#L71